code/index.html

```diff
@@ -1,22 +1,125 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Crafty Ideas E-zine</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Kalam&family=Kaushan+Script&family=Love+Ya+Like+A+Sister&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <div class="container">
+
+      <div class="header-container">
+
+      <header class="header">
+
+
+        <img class="logo" src="./logo.svg" alt="Technigo logo">
+        
+        
+        <h1 class="title-name">
+          Crafty Ideas 
+        </h1> 
+
+
+        <h1 class="title-type">
+          E-zine
+        </h1>
+
+
+        <a class="navbar navlink1" href="https://www.technigo.io/">Home</a> 
+
+
+        <a class="navbar navlink2" href="https://www.instagram.com/polkapetunian/">About</a> 
+
+
+        <a class="navbar navlink3" href="https://www.imdb.com/title/tt0118884">Contact</a> 
+
+
+        <a class="navbar navlink4" href="https://www.ravelry.com/account/login">Friends</a>
+        
+
+      </header>
+
+    </div>
```

Comment:
I'm guessing you've added some linebreaks here to improve readability, but let's trust correct indentation to improve our readability. Also, there should only be one `h1` of a site, think of it like the title of a book. You can use `span` if you want to style differently:
```
<main class="container">
  <div class="header-container">
    <header class="header">
      <img class="logo" src="./logo.svg" alt="Technigo logo">
      <h1 class="title-name">
        Crafty Ideas
        <span class="title-type">
        E-zine
        </span>
      </h1>
      <nav>
        <a class="navbar navlink1" href="https://www.technigo.io/">Home</a>
        <a class="navbar navlink2" href="https://www.instagram.com/polkapetunian/">About</a>
        <a class="navbar navlink3" href="https://www.imdb.com/title/tt0118884">Contact</a>
        <a class="navbar navlink4" href="https://www.ravelry.com/account/login">Friends</a>
      </nav>
    </header>
  </div>
```

Suggestion 👆´<main>´ and ´<nav>´